package/samplers/catcma/README.md

```diff
@@ -7,6 +7,13 @@ optuna_versions: [3.6.1]
 license: MIT License
 ---
 
+## Abstract
+
+The cutting-edge evolutionary computation algorithm CatCMA has been published on OptunaHub.
+CatCMA is an algorithm that excels in mixed search spaces with continuous and discrete variables.
+
+![The performance comparison results of CatCMA and other algorithms from https://arxiv.org/abs/2405.09962](images/catcma-performance.png)
```

Comment:
The reference is not rendered on the detailed page, and users cannot find it.
<img width="1120" alt="image" src="https://github.com/user-attachments/assets/ab016bd1-f112-4781-a373-86ac7afdb20e">